modules/service-esm.sh

```diff
@@ -1,6 +1,8 @@
 # shellcheck disable=SC2039
 
 ESM_SUPPORTED_SERIES="precise"
+ESM_SUPPORTED_ARCHS=""
```

Comment:
Also, I don't think the case of an empty list is tested. You could write a test that sets the arch to something exotic and that esm is still enabled (or using the new ALL special name, if you decide to accept that suggestion).

Reply:
added a test